packages/flame/lib/src/game/game_render_box.dart

```diff
@@ -58,11 +57,6 @@ class GameRenderBox extends RenderBox with WidgetsBindingObserver {
     markNeedsPaint();
   }
 
-  @override
-  void performLayout() {
```

Comment:
don't you still need to update the size variable when flutter changes our size?

Reply:
That's what I thought at first too, but it turns out https://github.com/flame-engine/flame/blob/main/packages/flame/lib/src/game/game_widget/game_widget.dart#L319 already does that.